lib/algolia/analytics.rb

```diff
@@ -0,0 +1,77 @@
+module Algolia
+
+  class Analytics
+    attr_reader :ssl, :ssl_version, :headers
+
+    def initialize(client, params)
+      @client = client
+
+      @headers      = params[:headers]
+      @ssl          = params[:ssl].nil? ? true : params[:ssl]
+      @ssl_version  = params[:ssl_version].nil? ? nil : params[:ssl_version]
+    end
+
+    def get_ab_tests(params = {})
+      params = {
+          :offset => 0,
+          :limit => 10,
+      }.merge(params)
+
+      perform_request(:GET, Protocol.ab_tests_uri, params)
+    end
+
+    def get_ab_test(ab_test_id)
+      raise ArgumentError.new('ab_test_id cannot be empty') if ab_test_id.nil? || ab_test_id == ''
```

Comment:
Maybe `ab_test_id.empty?` instead of `''`?

Reply:
It doesn't work because ab_test_id is normally a `fixnum`